extension/src/fileSystem/index.ts

```diff
@@ -199,16 +200,30 @@ stages:
   return appendFileSync(dvcYamlPath, pipeline)
 }
 
+export const addPlotToDvcYamlFile = (cwd: string, plotObj: PlotConfigData) => {
+  const dvcYamlFile = `${cwd}/dvc.yaml`
+  const { dataFile, ...plot } = plotObj
+  const plotName = relative(cwd, dataFile)
+  const dvcYamlLines = readFileSync(dvcYamlFile, 'utf8').split('\n')
+  const plotYaml = yaml.stringify({ plots: [{ [plotName]: plot }] }).split('\n')
+
+  // TBD this only works correctly for yaml with 2 space indent and no plots
+  // will adjust for other possibilities in another pr
```

Comment:
Taken care of in #4628 (still has a couple bugs I need to fix).

Reply:
[C] Let's not leave comments TODO in the code/merge before fixing obvious/known issue. It would have been enough to add a `do not merge` label to this PR and leave a comment linking to #4628 instead.

Reply:
Makes sense! Removed the unneeded comments.